_docs/aws_resources.md

```diff
@@ -25,8 +25,40 @@ After you have signed up for an AWS account and begin using services, keep in mi
-  
-The list above should be sufficient to get you started. You **do not** have to follow every step. However, pay close attention to **1, 3, and 5** because those are some of the most common solutions. 
 
-For additional information, AWS has a [comprehensive guide](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/checklistforunwantedcharges.html) to help you avoid unexpected charges if you find yourself in an uncharted territory. 
+The list above should be sufficient to get you started. You **do not** have to follow every step. However, pay close attention to **1, 3, and 5** because those are some of the most common solutions.
 
+For additional information, AWS has a [comprehensive guide](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/checklistforunwantedcharges.html) to help you avoid unexpected charges if you find yourself in an uncharted territory.
+
+
+
+# Video Tutorials To Help You Get Familiar With AWS Cloud services
+
+## Static Website Hosting on S3, Route 53 Domain Name Registration
+In this AWS S3 tutorial the presenter walks step-by-step through the process of hosting a website on Amazon S3. Then set up a custom domain name in AWS Route 53 and associate it with our website in S3
+(Domain registration is not free but at an affordable price. )
+
+[S3 Video Tutorial Available Here ] (https://www.youtube.com/watch?v=P_B79xXf-w8)
+
+## AWS Lambda & DynamoDB | AWS Serverless Tutorial
+This tutorial as uses AWS Lambda, AWS API Gateway and DynamoDB to create a serverless backend for your application. This is a three
+part series that can be accessed on "Cloud Path" via youTube.
+
+[Lambda DynamoDB Video Tutorial Available Here] (https://www.youtube.com/watch?v=VGerk8hrP9U)
+[Reserved Words In DynamoDB] (https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ReservedWords.html)
+
+## AWS Lamda Tutorial | Use AWS CloudWatch Events to Schedule a Lambda Function Invocation
+In this AWS Lambda tutorial we set up a basic hello world Lambda function that's triggered by a scheduled CloudWatch event to run every minute. This video is part of the AWS Serverless tutorial series.
+
+[Lambda & CloudWatch Video Tutorial Available Here.](https://www.youtube.com/watch?v=-v4LMV5DAD4&list=PLD_RqipW0-9s-u1HXTglYV8Aam-5P3XLi&index=7&t=0s)
+
+## How to Create EC2 Instance in AWS Step-by-step
+Create Your EC2 Resources and Launch Your EC2 Instance.
+How to create EC2 instance in AWS step by step.
+
+[EC2 Video TutorialAvailable here](https://www.youtube.com/watch?v=1x5dWRaLvIw)
+
+## AWS EC2 Linux Launch & Connection Using SSH
+
+[EC2 Linux Launch Video Available Here] (https://www.youtube.com/watch?v=v0g1M5bb9u4)
+[EC2 connect with Putty(Windows)] (This video will show how to use a PuTTY private key to connect to your Amazon EC2 Linux instance.)
```

Comment:
[EC2 connect with Putty(Windows)] (https://www.youtube.com/watch?v=bi7ow5NGC-U)
     This video will show how to use a PuTTY private key to connect to your Amazon EC2 Linux instance.